### ISSUE TITLE
when i start versions above 1.8 its starts launching pre release games isnstead of lunar

### ISSUE THREAD

**Issue by SaminYT** [Nilsen84/lunar-client-qt#248]

**Comment by gun07** [comment]
just update your minecraft through official lunar client launcher, then launch through LCQT.
then it should work.

**Comment by vidhu123yto** [comment]
did it worked ??

**Comment by gun07** [comment]
> did it worked ??

for me, it worked.
launch game with official launcher for once for update,
and launch with lcqt

**Comment by theghoulish76** [comment]
> _No description provided._

Yeah what @gun07 is saying works. Just delete the Lunar Client Qt folder, go into the Official Lunar Client, Launch your desired version, spend like 2 minutes doing something on it (maybe changing the settings and stuff), exit it, and open Lunar Client Qt and then launch that version. It should work now—It just worked for me.

**Comment by vidhu123yto** [comment]
No it's not working for me pls fix
